backend/src/main/java/de/tk/mastermind/repositories/PlayerRepository.java

```diff
@@ -0,0 +1,12 @@
+package de.tk.mastermind.repositories;
+
+import de.tk.mastermind.models.Player;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface PlayerRepository extends MongoRepository<Player, String> {
+    Optional<Player> findById(String id);
```

Comment:
You don't need to define that method. It is already present in the `MongoRepository`.